lib/danger/plugin_support/plugin_parser.rb

```diff
@@ -50,20 +48,24 @@ def to_dict(classes)
       end
 
       classes.map do |klass|
+        # Adds the class being parsed into the ruby runtime
+        require klass.file
+        real_klass = Danger.const_get klass.name
+        attribute_meths = klass.attributes[:instance].values.map { |v| v.values }.flatten
+
         {
-        name: klass.name.to_s,
-        body_md: klass.docstring,
-        example_code: klass.tags.select { |t| t.tag_name == "example" }.map(&:text).compact,
-        attributes: klass.attributes[:instance].map do |pair|
-          {
-          pair.first => d_attr.call(pair.last)
+          name: klass.name.to_s,
```

Comment:
This isn't a `Class` object - it's a Documentation object referring to the class